src/main/java/org/springframework/data/couchbase/core/convert/CouchbasePropertyValueConverterFactory.java

```diff
@@ -18,26 +18,28 @@
 
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
-import java.util.HashMap;
-import java.util.Map;
+import java.lang.reflect.Method;
 
 import org.springframework.data.convert.PropertyValueConverter;
 import org.springframework.data.convert.PropertyValueConverterFactory;
 import org.springframework.data.convert.ValueConversionContext;
+import org.springframework.data.couchbase.core.convert.translation.JsonValueConverter;
 import org.springframework.data.mapping.PersistentProperty;
 
 import com.couchbase.client.core.encryption.CryptoManager;
 import com.couchbase.client.java.encryption.annotation.Encrypted;
+import com.fasterxml.jackson.annotation.JsonValue;
 
 /**
- * Accept the Couchbase @Encrypted annotation in addition to @ValueConverter
+ * Accept the Couchbase @Encrypted annotation in addition to @ValueConverter.<br>
+ * Note that valueConversions.afterPropertiesSet() encapsulates this in a CachingPropertyValueConverterFactory making
+ * caching in this class unnecessary.
  *
  * @author Michael Reiche
  */
 public class CouchbasePropertyValueConverterFactory implements PropertyValueConverterFactory {
 
 	CryptoManager cryptoManager;
-	Map<Class<? extends PropertyValueConverter<?, ?, ?>>, PropertyValueConverter<?, ?, ?>> converterCache = new HashMap<>();
 
```

Comment:
don't need caching in our converter factory as it is encapsulated in a CachingPropertyValueConverterFactory